src/components/preview/index.tsx

```diff
@@ -0,0 +1,7 @@
+interface IPreview {
+  previewText: string;
+}
+
+export const Preview: React.FC<IPreview> = ({ previewText }) => {
+  return <div>{previewText}</div>;
+};
```

Comment:
Mudanças de styleguide
componentes:

`function() Componente {}
`
Definimos os componentes como function para que fique diferente do que é uma função do componente, do que é um coponente.

Funçoes
`const func () => {}
`

E vamos evitar o destructuring de props.

```suggestion
function Preview(props): React.FC<IPreviewProps> {
  return <div>{props.previewText}</div>;
};
```

Reply:
esse preview é de outro pr que ja foi aprovado, o certo seria eu abrir uma nova issue pra fazer essas modificações?